docs/features/app-groups.md

```diff
@@ -0,0 +1,9 @@
+---
+meta:
+  - name: description
+    content: "Fairwinds Insights | Documentation: About the App-Groups."
+---
+
+# App-Groups
```

Comment:
these won't show up until we add them to the .vuepress config, IIUC

Reply:
Interesting, I've created it because I'm linking to it [here](https://github.com/FairwindsOps/insights-docs/pull/385/files#diff-7446c7b41e269c90192fcea36fec1bb9e812a93c23951939922cc6a93b1893f4R134)

Should I remove it for now or add it to `.vuepress`?